src/HaskellWorks/Data/Uri/Location.hs

```diff
@@ -24,11 +24,13 @@ import qualified System.FilePath  as FP
 import qualified Data.Aeson.Types as J
 
 class IsPath a s | a -> s where
-  (</>) :: a -> s -> a
-  (<.>) :: a -> s -> a
+  (</>)  :: a -> s -> a
+  (<.>)  :: a -> s -> a
+  (-<.>) :: a -> s -> a
```

Comment:
This operator deserves docs I think 😁

Reply:
Is `-<.>` exactly the same as `<.>`?

Reply:
Oh, right.  It makes sense now.  Most instance implementations just delegate so they looks the same.

Reply:
I think the docs are implicit from seeing this in another path library and wanting to use it here